go/ct/gen/state_test.go

```diff
@@ -234,6 +234,20 @@ func TestStateGenerator_NonConflictingGasAreAccepted(t *testing.T) {
 	}
 }
 
+func TestStateGenerator_AddGasLowerUpperBoundIsEnforced(t *testing.T) {
+	generator := NewStateGenerator()
+	generator.AddGasLowerBound(42)
+	generator.AddGasUpperBound(44)
+
+	state, err := generator.Generate(rand.New(0))
+	if err != nil {
+		t.Fatalf("unexpected error during build: %v", err)
```

Comment:
for the future:
error strings are better written "failed to do something, the reason is...." 
This case: `failed to generate state, unexpected error ....`